Assets/MixedRealityToolkit/Providers/BaseSpatialAwarenessObserver.cs

```diff
@@ -96,7 +95,7 @@ public override int GetHashCode()
 
         private GameObject observedObjectParent = null;
 
-        public GameObject ObservedObjectParent => observedObjectParent ?? (observedObjectParent = SpatialAwarenessSystem?.CreateSpatialAwarenessObjectParent(Name));
+        public GameObject ObservedObjectParent => observedObjectParent != null ? ObservedObjectParent : (observedObjectParent = SpatialAwarenessSystem?.CreateSpatialAwarenessObjectParent(Name));
```

Comment:
are you suggesting this as the new pattern globally within MRTK?

Reply:
The difference here is that most of those patterns are referring to interfaces, while this one is a `GameObject`